packages/cli/test/dev/integration-3.test.ts

```diff
@@ -70,27 +70,6 @@ test(
     await testPath(200, '/', /React App/m);
   })
 );
-/*
-test(
-  '[vercel dev] 05-gatsby',
-  testFixtureStdio('05-gatsby', async (testPath: any) => {
-    await testPath(200, '/', /Gatsby Default Starter/m);
-  })
-);
-*/
-test(
-  '[vercel dev] 06-gridsome',
-  testFixtureStdio('06-gridsome', async (testPath: any) => {
-    await testPath(200, '/');
-    await testPath(200, '/about');
-    await testPath(308, '/support', 'Redirecting to /about?ref=support (308)', {
-      Location: '/about?ref=support',
-    });
```

Comment:
Gridsome was outdated and there's already a test for redirects here: 

https://github.com/vercel/vercel/blob/ad107ecf79a15082dff683855e066d0b251e9127/packages/cli/test/dev/integration-2.test.ts#L91